app/models/user.rb

```diff
@@ -1,9 +1,13 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  #  :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
- [optional] please kindly remove the comments to keep your work clean .. nice work